info.olasagasti.revelation.json

```diff
@@ -0,0 +1,71 @@
+{
+    "app-id": "info.olasagasti.revelation",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.36",
+    "sdk" : "org.gnome.Sdk",
+    "rename-desktop-file" : "revelation.desktop",
+    "rename-icon" : "revelation",
+    "rename-icon" : "revelation-locked",
+    "command" : "revelation",
+    "finish-args" : [
+        "--share=ipc",
+        "--share=network",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--filesystem=xdg-run/gvfs",
+        "--filesystem=/tmp",
+        "--filesystem=home",
```

Comment:
Does it need full home access?

Reply:
Okay, if this is about opening/saving files, `GtkFileChooserNative` should have access everywhere without poking holes in the sandbox. `VFS` permissions could be dropped then too.

Reply:
I'll need some time to port GtkFileChooserDialog to GtkFileChooserNative. Is this a hard requirement or could be changed later?